doc/source/cli.md

```diff
@@ -0,0 +1,407 @@
+(cli)=
+
+# Command Line Interface (CLI)
+
+To simplify the use of xDEM and provide a universal tool to assess one or multiple DEMs,
+we have decided to implement a Command Line Interface (CLI).
+To support this, we offer a set of workflows that can be easily run using a configuration file.
+Users can also create their own workflows and submit them for inclusion.
+
+```{note}
+**workflow definition** : combinations of various xDEM and GeoUtils features tailored to specific applications
+```
+## Global information
+
+All workflows follow the same command structure:
+```{code}
+xdem workflow_name --config config_file.yaml
+```
+
+The configuration YAML file contains at least the required input parameters for the workflow.
+xDEM then automatically fills in the rest with suggested settings. Users are free to edit the
+configuration to run only the parts they need.
+
+
+It is possible to display a template of all possible configuration options for the file using the following command:
+
+```{code}
+xdem workflow_name --generate-config
+```
+
+```{note}
+At the end of the execution, several output files are saved to disk, including an HTML report
+and its corresponding PDF version.
+```
+
+
+## Workflow diff-analysis
+
+### Summary
+
+The diff-analysis workflow allows the user to generate various outputs to facilitate
+the analysis of differences between two elevation datasets. An optional coregistration step can also be included.
+
+:::{figure} imgs/diffanalysis_pipeline.png
```

Comment:
How did you generate these plots?
If this needs manual steps, you could write a xDEM-Wiki page on how to do so, so we can add future workflows the same way :wink:

Reply:
It's really a great idea to summarize the workflows visually. :slightly_smiling_face: 
Once we've converged to a final workflow on this PR (can wait for now), we can try to minimize blank space, increase font size, and homogenize the namings for the final rendering. :smile: